openwisp_monitoring/device/schema.py

```diff
@@ -1,7 +1,10 @@
 # NetJSON DeviceMonitoring schema,
 # https://github.com/netjson/netjson/blob/master/schema/device-monitoring.json
 schema = {
-    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/netjson/netjson/master/schema/device-monitoring.json",
```

Comment:
one question: do you know if this triggers the downoad of this file? or is it there just for documentation purposes?